MongoTrait/TrashCanable.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace OpenOrchestra\MongoTrait;
+
+/**
+ * Class TrashCanable
+ */
+trait TrashCanable
```

Comment:
SoftDeleteable ?

It will be more close to the name from the doctrine behaviour